client/src/components/pages/members/memberInputs.js

```diff
@@ -1,5 +1,3 @@
-import moment from 'moment';
-
 const memberInputs = [
     {
         id: 'Personal Info',
```

Comment:
the id should not be the display name. The id should be `personalInfo`. This is because we may not want to display a title for a section, and this isn't allowing that.